### DISCUSSION TITLE
Can I just release the bin file compiled by this GitHub Action?

### DISCUSSION THREAD

**Discussion by Love4yzp** [espressif/esp-idf-ci-action#32] [General]
I've successfully to compiled the bin file, I wish to automatically package it and release it.
Can it possible?

**Comment by kumekay** [comment]
Hello @Love4yzp 

Do you want to make a GitHub release? If so, it's possible. However, it's out of the scope of this action, you can use one of these actions for the task:

- https://github.com/softprops/action-gh-release
- https://github.com/ncipollo/release-action
